src/app/selects/select-static.html

```diff
@@ -0,0 +1,151 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<form class="test-select">
+    <section class="form-block">
+        <label>Select Boxes</label>
+        <div class="form-group">
+            <label for="selects_1">This is a select box</label>
+            <div class="select">
+                <select id="selects_1">
+                    <option>1</option>
+                    <option>2</option>
+                    <option>3</option>
+                    <option>4</option>
+                    <option>50000000</option>
+                </select>
+            </div>
+        </div>
+        <div class="form-group">
+            <label for="selects_2">Text field paired with a select</label>
+            <input type="text" id="selects_2" placeholder="Test">
+            <div class="select">
+                <select id="selects_3">
+                    <option>New York</option>
+                    <option>San Francisco</option>
+                    <option>Austin</option>
+                    <option>Paris</option>
+                    <option>Seattle</option>
```

Comment:
Thank you for not putting Jersey City in there. 👍

Reply:
Just because of this I will make sure I add it in there. 😝